lib/backend/k8s/resources/customresource.go

```diff
@@ -103,9 +103,12 @@ func (c *customK8sResourceClient) Create(kvp *model.KVPair) (*model.KVPair, erro
 		return nil, K8sErrorToCalico(err, kvp.Key)
 	}
 
-	// Update the revision information from the response.
-	kvp.Revision = resOut.GetObjectMeta().GetResourceVersion()
-	return kvp, nil
+	// Return the Key and Value with updated Revision information.
```

Comment:
Well we don't really need it although it makes the testing easier.   Reasoning is, if you call

kvp_out, err := client.Create(kvp_in)

You wouldn't expect kvp_in to change, and therefore you might wish to use it again.  But as it stands we'll update kvp_in and just return it as kvp_out.

Anyhow, I've removed as it wasn't needed for this PR.